src/snowflake/cli/plugins/cortex/commands.py

```diff
@@ -21,8 +28,56 @@
 
 app = SnowTyper(
     name="cortex",
-    help="Provides access to Snowflake Cortex LLM.",
+    help="Provides access to Snowflake Cortex.",
+)
+
+search_command_enabled = sys.version_info < (3, 12)
+
+
+@app.command(
+    requires_connection=True,
+    hidden=not search_command_enabled,
 )
+def search(
+    query: str = typer.Argument(help="The search query string"),
+    service: str = typer.Option(
+        help="Cortex search service to be used. Example: --service my_cortex_service",
+    ),
+    columns: Optional[List[str]] = typer.Option(
+        help='Columns that will be returned with the results. If none is provided, only search column will be included in results. Example --columns "foo" --columns "bar"',
+        default=None,
+    ),
+    limit: int = typer.Option(help="Maximum number of results retrieved", default=1),
+    **options,
+):
+    """
+    Performs query search using Cortex Search Services.
+    """
+
+    if not search_command_enabled:
```

Comment:
```suggestion
    if not sys.version_info < (3, 12):
```
WDYT? Do we use this variable in more than one place?

Reply:
It's a hack for tests - if i mock the `sys.version_info` - tests fail, because of the inner workings of the typer (it chooses the options for python 3.12 using python 3.8 in tests).
But if i mock the result of this comparison - everything is ok.

Reply:
Then at least let's make this constant upper case 👌

Reply:
Done